File: libs/eave-stdlib-ts/src/core-api/models/github.ts
export type GithubInstallationInput = {
  github_install_id: string;
};

export type GithubInstallation = {
  id: string;
  /** eave TeamOrm model id */
  team_id: string;
  github_install_id: string;
};

// Typealias for naming consistency
export type GithubInstallationPeek = GithubInstallation;

File: apps/core/eave/core/graphql/types/github.py
import uuid

import strawberry
from eave.stdlib.core_api.models import BaseInputModel, BaseResponseModel

@strawberry.type
class GithubInstallation(BaseResponseModel):
    id: uuid.UUID
    team_id: uuid.UUID
    """eave TeamOrm model id"""
    github_install_id: str


GithubInstallationPeek = GithubInstallation
"""Type Alias for naming consistency with other integrations."""

@strawberry.input
class GithubInstallationInput:
    github_install_id: str
